community/cypher/cypher/src/test/scala/org/neo4j/cypher/SemanticErrorAcceptanceTest.scala

```diff
@@ -666,6 +666,18 @@ class SemanticErrorAcceptanceTest extends ExecutionEngineFunSuite {
       "A single relationship type must be specified for MERGE (line 1, column 9 (offset: 8))")
   }
 
+  test("give a nice error message when missing crs in cartesian point") {
+    executeAndEnsureError("RETURN point({x: 2.3, y: 4.5}) as point",
+                          "A cartesian point must contain a 'crs' (coordinate reference system) (line 1, column 14 (offset: 13))")
```

Comment:
Is that how you see it, is the names of the coordinates or the crs the most important thing? Before this change all we got was a `java.util.NoSuchElementException`?

I would think if a user put in an x and a y there the intention is to use a cartesian point not a geographic point don't you think?